README.md

```diff
@@ -280,6 +303,7 @@ storm::config_map:
     - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
       parallelism.hint: 1
       argument:
+        metrics.reporter.name: "com.verisign.storm.metrics.reporters.GraphiteReporter"
```

Comment:
Should we consider renaming `GraphiteMetricsConsumer` now that we support more than one destination backend?